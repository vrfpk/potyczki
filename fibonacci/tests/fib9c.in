58777403136344064
