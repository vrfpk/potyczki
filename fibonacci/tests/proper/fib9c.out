217449356558057472
